code/segmentedtickerplant/stplog.q

```diff
@@ -221,6 +223,8 @@ endofday:{[date;data]
 
 // STP runs function to send out eod messages and roll logs
 stpeod:{[date;data]
+  .lg.o[`endofday;"flushing remaining data to subscribers and clearing tables"];
+  .stpps.pubclear[.stplg.t];
```

Comment:
I think this effectively makes `stpeod` functionally identical to `endofday` - should we just eliminate `stpeod` and replace any references to it with `endofday` instead?